tests/std/tests/P0896R4_ranges_alg_mismatch/test.cpp

```diff
@@ -52,7 +48,7 @@ constexpr void smoke_test() {
         assert((*result.in2 == std::pair<long, long>{13, 5}));
     }
     {
-        // Validate sized iterator+sentinel pairs
+        // Validate sized iterator + sentinel pairs
         auto result = mismatch(x.begin(), x.end(), y.begin(), y.end(), equal_to{}, get_first, get_second);
         using I     = iterator_t<R const>;
         using O     = std::array<std::pair<long, long>, 3>::const_iterator;

```

Comment:
There are occurrences of `using O` in this test that appear to be relics of the "in_out" mistake.